scripts/checkpatch.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+SCRIPT=$(readlink -f $0)
+SCRIPT_PATH=$(dirname "$SCRIPT")
+CHECKPATCH=$SCRIPT_PATH/../../linux/scripts/checkpatch.pl
```

Comment:
this is quite ugly in the optee_os.
Couldn't caller insure `checkpatch.pl` is reachable from the PATH.
your script could just check whether exec is found.